packages/jobs/lib/crons/autoIdleDemo.ts

```diff
@@ -0,0 +1,128 @@
+import { schedule } from 'node-cron';
+import {
+    CommandToActivityLog,
+    ErrorSourceEnum,
+    ScheduleStatus,
+    SyncClient,
+    SyncCommand,
+    SyncConfig,
+    createActivityLog,
+    createActivityLogMessageAndEnd,
+    db,
+    errorManager,
+    updateSuccess as updateSuccessActivityLog,
+    updateScheduleStatus,
+    isErr
+} from '@nangohq/shared';
+import tracer from 'dd-trace';
+
+export async function cronAutoIdleDemo(): Promise<void> {
+    schedule('1 * * * *', () => {
+        const span = tracer.startSpan('cron.syncs.idleDemo');
+        tracer.scope().activate(span, async () => {
+            try {
+                await exec();
+            } catch (err: unknown) {
+                const e = new Error('failed_to_auto_idle_demo', { cause: err instanceof Error ? err.message : err });
+                errorManager.report(e, { source: ErrorSourceEnum.PLATFORM }, tracer);
+            }
+            span.finish();
+        });
+    });
+}
+
+interface ResQuery {
+    id: string;
+    name: string;
+    environment_id: number;
+    provider: string;
+    connection_id: number;
+    unique_key: string;
+    schedule_id: string;
+}
+async function exec() {
+    console.log('[autoidle] starting');
+
+    const q = db.knex
```

Comment:
looks like this query should be in `sync.service` where all the syncs model related functions are. no?

Reply:
fixed in [e6c2681](https://github.com/NangoHQ/nango/pull/1631/commits/e6c2681df513185d4e4f39587c7afb7980aa8557)